samples/meetings-notification/nodejs/views/index.html

```diff
@@ -26,7 +26,7 @@
                     myValue: document.forms["contentForm"]["option"].value,
                     title: "<%= question %>"
                 }
-                microsoftTeams.tasks.submitTask(contentInfo, "<<APP_ID>>");
+                microsoftTeams.tasks.submitTask(contentInfo, "daecec22-636b-470f-ab26-c60de025cde6");
```

Comment:
remove this id, it should not come from client side

Reply:
fixed